src/utils/versioning.js

```diff
@@ -9,9 +9,10 @@ const SUPPORTED_VERSIONS = [
   'v0.38',
   'v0.39',
   'v0.40',
+  'v0.41',
```

Comment:
Maybe we could also clean up a few old versions, WDYT? :thinking:

Reply:
What's the policy here, I also thought we should remove some of them, but how many? We should also [document it](https://github.com/grafana/k6-docs/wiki/Add-version-for-Javascript-API-documentation) to make it part of the release process. I think for this time I would prefer to open an issue, sync with the team and address it in the next few days.

Reply:
https://github.com/grafana/k6-docs/issues/955